CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Version 1.0.1-beta.1 (2019-05-22)
```

Comment:
I think we should stick to 1.0 till we're out of beta. So this would be 1.0-beta.2.